webservices/rest.py

```diff
@@ -165,15 +165,16 @@ def add_caching_headers(response):
 @app.errorhandler(Exception)
 def handle_exception(exception):
     wrapped = ResponseException(str(exception), ErrorCode.INTERNAL_ERROR, type(exception))
-
+    requested_url = request.url
```

Comment:
Do we need this variable?

Reply:
Thought of logging the URL so we know which one failed.

Reply:
I am not sure I follow, it is not being used in the following logging call. It is used in one place, which is:
```
raise exceptions.ApiError('Could not process the request %s'.format(requested_url),
            status_code=http.client.NOT_FOUND)
```
but that could use `request.url` directly, right?

Reply:
either way is fine i think.

Reply:
I disagree. Sometimes, it makes sense to create a variable to hold a field from an object, when the name adds more clarity, especially when the value is being used in multiple places. In this case, we are adding code with no payoff.